compiler.js

```diff
@@ -184,28 +374,21 @@ class SlangCompiler
                 return null;
             }
 
-            var module = slangSession.loadModuleFromSource(shaderSource, "user", "/user.slang");
-            if(!module) {
+            // Load the user module, this is the source code that user inputs in the editor
+            var userModule = slangSession.loadModuleFromSource(shaderSource, "user", "/user.slang");
+            if(!userModule) {
                 var error = this.slangWasmModule.getLastError();
                 console.error(error.type + " error: " + error.message);
                 this.diagnosticsMsg+=(error.type + " error: " + error.message);
                 return null;
             }
 
             var components = new this.slangWasmModule.ComponentTypeList();
-            components.push_back(module);
+            components.push_back(userModule);
```

Comment:
did we make sure to delete the component list and everything pushed to it?

Reply:
ack